src/glean.js

```diff
@@ -26,3 +25,5 @@ class Glean {
         glean._eventStorage.record(Date.now(), "window", "onload", { "what": "it-works" })
     });
 })();
+
+window.Glean = Glean
```

Comment:
Why do we need this? And, even if we needed, did you want the instance here?

Reply:
Oops, that shouldn't have been left there. It's from when I tested it.